index.css

```diff
@@ -44,40 +44,167 @@ header .logo {
   color: var(--secondary-color);
 }
 
-header img {
+.top-level img {
   height: 12px;
   align-self: center;
 }
 
 main {
   background: white url(./images/mobile-bg.png) center/contain no-repeat;
-  height: calc(100vh - 43px);
   border-bottom-left-radius: 5rem;
+  height: 626px;
+  margin-bottom: 114px;
+}
+
+.banner {
   width: 100%;
-  padding: 80px 24px 138px 24px;
+  padding: 114px 24px 138px 24px;
   background-color: white;
 }
 
-main .intro {
+.banner .intro {
   font-size: 40px;
-  font-weight: 700;
+  font-weight: var(--bold-font);
 }
 
-main .dev {
+.banner .dev {
   font-weight: var(--regular-font);
   margin-top: 12px;
 }
 
-main h4 {
+.banner h4 {
   font-weight: var(--medium-font);
   text-transform: uppercase;
   margin-top: 12px;
   color: var(--secondary-color);
 }
 
-main ul {
+.banner ul {
   display: flex;
   margin-top: 12px;
   list-style: none;
   gap: 20px;
 }
+
+.grid-container {
+  margin: 0 24px;
+}
+
+.skills {
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  margin: 0 0 88px 0;
+  border: solid 1px #dfe1e6;
+  border-radius: 16px;
+  background-color: white;
+}
+
+.skills .sample {
+  max-width: 544px;
+}
+
+.project-header h2 {
+  font-weight: var(--bold-font);
+  font-size: 32px;
+  text-transform: capitalize;
+  margin: 12px auto;
+}
+
+.tags {
+  display: flex;
+  gap: 20px;
+  margin-bottom: 20px;
+}
+
+.tags li {
+  font-size: 13px;
+  font-weight: var(--semi-bold-font);
+  color: #7a869a;
+  margin-right: 0.5rem;
+  text-transform: capitalize;
+}
+
+.tags .diff-li {
+  text-transform: uppercase;
+  color: #344563;
+  list-style: none;
+}
+
+.project-header p {
+  font-weight: var(--regular-font);
+  color: var(--text-color);
+  font-size: 15px;
+  margin-bottom: 20px;
+}
+
+.set-stack {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 8px;
+}
+
+.set-stack li {
+  list-style: none;
+  margin-bottom: 20px;
+}
+
+.stack {
+  background-color: var(--pale-secondary-bg);
+  padding: 4px 12px;
+  color: var(--secondary-color);
+  font-size: 12px;
+  border-radius: 8px;
+}
+
+.btn {
+  text-transform: capitalize;
+  font-weight: var(--medium-font);
+  font-size: 17px;
+  padding: 16px 12px;
+  border: 1px solid var(--btn-color);
+  color: var(--btn-color);
+  background-color: white;
+  border-radius: 8px;
+  margin-bottom: 20px;
+  cursor: pointer;
+}
+
+.btn:hover {
+  background-color: #6070ff;
+  color: white;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+}
+
+.btn:active {
+  color: white;
+  background-color: var(--btn-color);
+}
+
+.about-myself {
+  border-top-right-radius: 5rem;
+}
+
+.about-myself .btn {
+  margin-top: 20px;
+}
```

Comment:
- [ ] Kindly use CSS `flex-box` to the about-myself section in order to align your content correctly and it will help you to maintain the website responsiveness very easily :100: as it is one of the mandatory requirements in project requirements. I will even make our work look more professional :wink: